recipes/llama-cpp/all/cmake/llama-cpp-cuda-static.cmake

```
@@ -0,0 +1,7 @@
+find_package(CUDAToolkit REQUIRED)
```

```suggestion
include(CMakeFindDependencyMacro)
find_dependency(CUDAToolkit REQUIRED)
```
`find_dependency()` is preferable to `find_package()` in CMake config modules: https://cmake.org/cmake/help/latest/module/CMakeFindDependencyMacro.html